test/integration/testsuite/prepare.go

```diff
@@ -102,16 +102,14 @@ func determineIsoFromFile(isoUrl string) string {
 	return isoName
 }
 
-func setUp() string {
+func setupTestDirectory() string {
 	var err error
 	if testDir == "" {
 		testDir, err = ioutil.TempDir("", "minishift-integration-test-")
 		if err != nil {
 			fmt.Println("Error creating temporary directory:", err)
 			os.Exit(1)
 		}
-	} else {
-		ensureTestDirEmpty()
```

Comment:
@agajdosi I think this was a good check to make sure the directory is empty. In case some of the logic to clean up did not work, we can still make sure test should not run if the test directory is not empty.  IMO we should keep it. WDYT?

Reply:
@agajdosi Nevermind . my comment was premature.